### ISSUE TITLE
[Feature request] Support for custom Deluge Name after processing?

### ISSUE THREAD

**Issue by RonSijm** [Laharah/deluge-FileBotTool#54]
Hey there,

I'm using the latest version of deluge-FileBotTool, deluge, and FileBotTool.
I'm using this expression to process show seasons: `{n} [{y}]/Season {s} - {sy}/{n} - {s00e00} - {t.colon(' - ')}.{vf}`

This works fine when downloading individual episodes, though when I download an entire season, the name that's left in deluge is this:

<img width="666" alt="deluge" src="https://user-images.githubusercontent.com/337928/222896999-5682f33d-f91e-453b-96b3-2caefa585b83.png">

So basically the name in deluge is just `Season {s} - {sy}` - which makes it confusing to see what is what when I have multiple shows

It would be nice if in the profile or preset we could set a custom "new deluge name" to have more control over the Deluge naming

**Comment by Laharah** [comment]
Unfortunately this is a deluge limitation. Deluge uses the top level name for the torrent display title. Your options here are to:
- include the title in the season name (eg: `{n} - Season {s}`) 
- keep the original name and use symbolic or hard link mode.

It's a behavior I missed when I switched from uTorrent so I sympathize.
